src/SFA.DAS.ApplyService.InternalApi/Infrastructure/OuterApiClient.cs

```diff
@@ -1,38 +1,48 @@
 ﻿using AutoMapper;
-using CharityCommissionService;
 using Microsoft.Extensions.Logging;
 using SFA.DAS.ApplyService.Configuration;
 using System.Threading.Tasks;
 
 namespace SFA.DAS.ApplyService.InternalApi.Infrastructure
 {
+    using Newtonsoft.Json;
+    using SFA.DAS.ApplyService.InternalApi.Types.CharityCommission;
     using System;
+    using System.Net.Http;
 
     /// <summary>
     /// Charity Commission API docs are located at: http://apps.charitycommission.gov.uk/Showcharity/API/SearchCharitiesV1/Docs/DevGuideHome.aspx
     /// Charity Commission WSDL is located at: https://apps.charitycommission.gov.uk/Showcharity/API/SearchCharitiesV1/SearchCharitiesV1.asmx?WSDL
     /// There is a Web-Friendly version located at: http://beta.charitycommission.gov.uk/charity-search/
     /// </summary>
-    public class CharityCommissionApiClient
+    public class OuterApiClient
     {
-        private readonly ISearchCharitiesV1SoapClient _client;
-        private readonly ILogger<CharityCommissionApiClient> _logger;
+        private const string _acceptHeaderName = "Accept";
+        protected const string _contentType = "application/json";
+
+        private readonly HttpClient _client;
+        private readonly ILogger<OuterApiClient> _logger;
         private readonly IApplyConfig _config;
 
-        public CharityCommissionApiClient()
+        public OuterApiClient()
         {
-            // Constructor used for Mocking CharityCommissionApiClient
+            // Constructor used for Mocking CharityCommissionOuterApiClient
```

Comment:
Can we get rid of this constructor? We shouldn't have created this in first place.

Reply:
Done

Reply:
Default constructor is needed otherwise build is failing